sdk/core/azure-core/CMakeLists.txt

```diff
@@ -49,7 +49,9 @@ set(
     inc/azure/core/credentials/token_credential_options.hpp
     inc/azure/core/cryptography/hash.hpp
     inc/azure/core/diagnostics/logger.hpp
+    inc/azure/core/http/http_status_code.hpp
     inc/azure/core/http/http.hpp
+    inc/azure/core/http/raw_response.hpp
```

Comment:
We need to include these headers to `core.hpp` and add tests to the simplified header.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-cpp/pull/2070

Reply:
we don't need to. The headers are part of Http.hpp.

Reply:
Also, moving types in public headers to another is considered a breaking change. Why did we do it?

It needs a CL entry - fixed in https://github.com/Azure/azure-sdk-for-cpp/pull/2069

Reply:
> moving types in public headers to another is considered a breaking change

Not if the the type is moved to a new header (say header.hpp) and then the previous header includes header.hpp.  In that case, there's no change at all to customer.

A customer adding Http.hpp header before will continue to have same behavior.

I moved those types out of Http.hpp because they are required to be known by the RequestFailedException which at the same time is part of Http.hpp.  But removing exceotion.hpp to move RequestFailedException to Http.hpp would have been a breaking change.   And it makes sense for a customer to only include exception.hpp they only want to catch the RequestFailedException without adding the entire Http.hpp.